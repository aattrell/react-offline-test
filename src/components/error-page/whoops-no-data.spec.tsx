import React from "react";
import ErrorPage from "./whoops-no-data";
import { shallow } from "enzyme";

const expectedErrorMessage =
  "We don't have access to energy generation mix data at this time. Please check back later.";

it("renders the error page correctly", () => {
  const component = shallow(<ErrorPage />);

  expect(component.html()).toMatchSnapshot();
  expect(component.text()).toContain(expectedErrorMessage);
});
