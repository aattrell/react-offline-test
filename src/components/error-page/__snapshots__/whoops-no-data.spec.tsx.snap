// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the error page correctly 1`] = `"<div><h1>Whoops! Something has gone wrong</h1><p>We don&#x27;t have access to energy generation mix data at this time. Please check back later.</p></div>"`;
